server/src/main/java/org/jboss/as/server/ServerService.java

```diff
@@ -186,6 +187,8 @@ public final class ServerService extends AbstractControllerService {
     static final RuntimeCapability<Void> EXTERNAL_MODULE_CAPABILITY =
             RuntimeCapability.Builder.of(EXTERNAL_MODULE_CAPABILITY_NAME, ExternalModule.class)
                     .build();
+   public static final RuntimeCapability<Void> SERVER_ENVIRONMENT_CAPABILITY = RuntimeCapability.Builder.of(SERVER_ENVIRONMENT_CAPABILITY_NAME, ServerEnvironment.class).build();
```

Comment:
We are still planning to reduce the visibility of this, correct?

Reply:
Maybe I missing something but when you mentioned `OperationContext.getCapabilityServiceName("org.wildfly.server.environment")` I thought you mean to your capability.getCapabilityServiceName()

Reply:
I meant that OperationContext.getCapabilityServiceName("org.wildfly.server.environment", ServerEnvironment.class) can be used to generate a ServiceName for use with a standard ServiceTarget.

Reply:
`getCapabilityServiceName("org.wildfly.server.environment", ServerEnvironment.class)` is not the static method. We can't use it like this.

Reply:
Got it. Thanks